src/customJs/payuButtonTool/index.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { setLocale } from '../localization';
+import {PayuButtonTool} from "./PayuButtonTool";
+import * as React from "react"
+import {render, screen} from "@testing-library/react";
+
+describe(PayuButtonTool.name, ()=>{
+    it.each([
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'small', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'medium', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'large', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'small', lang: 'en-US'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'medium', lang: 'en-US'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'large', lang: 'en-US'}
+    ])("must be render with $size size and language $lang", ({baseUrl, size, lang})=>{
+        prepareUnlayerGlobalObject();
+        setLocale(lang);
+        const locale = {'es-ES': 'es','en-US': 'en'}[lang]
+
+        const expectedLink = `https://doppler.unlayer.editor/${locale}/${size}`
+        const values = {
+            size,
+            payu_detail: expectedLink,
+            alignment: 'center'
```

Comment:
If I am not wrong, the link has no special logic, and the _expected_ word confused me

```suggestion
        const arbitraryLink = 'https://test';
        const values = {
            size,
            payu_detail: arbitraryLink,
            alignment: 'center'
```